security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+Security is at the heart of a healthy and dependable software ecosystem and therefore also at the heart of the SSWG. This page intends to give clear and easy to follow information to users and developers of packages listed on the SSWG's package index.
+
+If you find any information to be missing, unclear, or misleading, please contact any member of the SSWG or create a post in the [server area](https://forums.swift.org/c/server/serverdev/14) of the Swift Forums.
+
+## I found a security vulnerability, what should I do?
+
+If you found a security vulnerability, first of all thank you for caring about our ecosystem. We rely on the community to keeping us all as safe and secure as we can (at least when it comes to software).
```

Comment:
@glbrntt +1 on suspected vulns. The repetition is deliberate because this page addresses different personas such as "I am a random person and found a security vuln, what should I do" as well as "I am a maintainer, what do I need to do when a vuln report happened". I think that's not clear that nobody is expected to read the whole doc but rather only the section that matches where they are right now.

Reply:
Regarding the repetition: Maybe we should have a security landing page + links like "I found a vulnerability, what should I do?" / "SSWG Security requirements for package authors" / "I am a package author, somebody reported a vuln, what's next"

Reply:
Yeah, landing page and separate files or just making the sections more obvious with dividers and a contents would also work.

Reply:
done